data/boltimpl.go

```diff
@@ -67,12 +67,15 @@ func LoadWorldFromDB(filename string) World {
 
 // UserData is a JSON-serializable set of information about a User.
 type UserData struct {
-	Gold        int
-	PylonAmount int
-	Username    string `json:""`
-	Location    UserLocation
-	Items       []Item
-	PrivKey     string
+	Gold             int
+	PylonAmount      int
+	Username         string `json:""`
+	Location         UserLocation
+	Items            []Item
+	DefaultItemIndex int
+	CHARs            []Item
```

Comment:
can you we something better then `CHARs` here?

Reply:
ok